test/pkg/environment/aws/expectations.go

```diff
@@ -413,20 +444,23 @@ func (env *Environment) GetDeprecatedAMI(amiID string, amifamily string) string
 	return lo.FromPtr(output.ImageId)
 }
 
-func (env *Environment) EventuallyExpectRunInstances(instanceInput *ec2.RunInstancesInput) *ec2.Reservation {
+func (env *Environment) EventuallyExpectRunInstances(instanceInput *ec2.RunInstancesInput) ec2types.Reservation {
 	GinkgoHelper()
 	// implement IMDSv2
-	instanceInput.MetadataOptions = &ec2.InstanceMetadataOptionsRequest{
-		HttpEndpoint: aws.String("enabled"),
-		HttpTokens:   aws.String("required"),
+	instanceInput.MetadataOptions = &ec2types.InstanceMetadataOptionsRequest{
+		HttpEndpoint: "enabled",
+		HttpTokens:   "required",
 	}
-	var out *ec2.Reservation
-	var err error
+	var reservation ec2types.Reservation
 	Eventually(func(g Gomega) {
-		out, err = env.EC2API.RunInstances(instanceInput)
+		runInstancesOutput, err := env.EC2API.RunInstances(env.Context, instanceInput)
```

Comment:
```suggestion
		out, err := env.EC2API.RunInstances(env.Context, instanceInput)
```

nit: seems like there's no reason for us to change this variable name